compiler/optimizer/OMRValuePropagation.hpp

```diff
@@ -445,6 +444,7 @@ class ValuePropagation : public TR::Optimization
    void removeRestOfBlock();
    void mustTakeException();
    void processTrees(TR::TreeTop *startTree, TR::TreeTop *endTree);
+   virtual void transformRecognizedMethod(TR::Node *node) {}
```

Comment:
I see. Please create this method as a part of that set of changes to make it logically complete.